charts/auth-layer-proxy/README.md

```diff
@@ -0,0 +1,50 @@
+# HederaTheGraph (HTG) Authentication Layer Proxy Helm Chart
+
+This chart deploys the HederaTheGraph (HTG) Authentication Layer Proxy to your Kubernetes cluster.
+Is based on the project [auth-layer-proxy](link) and is a proxy that adds authentication to the requests made to the TheGraph Admin API.
+
+## Prerequisites
```

Comment:
Repeated below.
Remove one instance of the heading

Reply:
✅  Done